package.json

```diff
@@ -230,6 +230,7 @@
     "lodash": "^4.17.15",
     "mac-ca": "^1.0.4",
     "marked": "^1.1.0",
+    "material-design-icons-iconfont": "^6.1.0",
```

Comment:
Why this is needed? We have embedded (downloaded) font already for material-icons (from latest master).
See `renderer/fonts/MaterialIcons-Regular.ttf`

Reply:
See my comment on the main description. That "latest master" is years old and out of date.

Reply:
Updated 2 months ago..
https://github.com/google/material-design-icons/tree/master/font

Reply:
Which icons are missing there ^?

Reply:
Well none the less, even that font is missing "open_in_full" and "close_fullscreen". So it might have been updated but it is still incomplete.

Reply:
They both available currently in master:
```
<Icon material={"open_in_full"}/>
<Icon material={"close_fullscreen"}/>
```
![image](https://user-images.githubusercontent.com/6377066/98263358-6332f380-1f8f-11eb-995f-98abfdf7a854.png)

Reply:
Okay I see what has happened. I started work on this 22 days ago, but you only updated to the latest version 12 days ago, so when I was looking into this we were behind.